code/__DEFINES/mobs.dm

```diff
@@ -458,3 +458,13 @@ var/list/global_mutations = list() // list of hidden mutation things
 #define MOTH_WINGS_BEHIND_LAYER	1
 
 #define TOTAL_UNDERLAYS			1
+
+
+//Ravager defines:
+#define CHARGESPEED					3
+#define CHARGESTRENGTH				3
+#define CHARGEDISTANCE				5
+#define CHARGECOOLDOWN				300
```

Comment:
Charge is pretty generic so you'll probably do better by adding RAV_ at the start.